source/contributors/becomingacommitter.md

```diff
@@ -0,0 +1,93 @@
+
+Code contributions are not the only type of contribution that counts towards becoming a committer, it’s just the most common. Design, documentation, marketing, event management, and many other ways of contributing to the success of a project are also often considered in making someone a committer. While the term “committer” implies committing code, it can also be interpreted as someone who is committed to the project.
+
+# End user support
+Answering end user questions has many benefits. It’s the best way to establish expertise in aspects of the software that effect actual users, and, thus, the best way to stay in touch with user concerns. It’s also the way to establish and maintain visibility in the project community, because your name is always visible on the mailing list.
+
+Caution: Do not just jump in and answer every question with visibility as a primary goal. Ensure that your answers are actually useful, and contribute something to the conversation. Simply posting to every conversation, particularly when someone else has already offered a good answer, can be perceived as attempting to game the metrics.
+
+# Documentation
+Improving the documentation is one of the most effective things that one can do to improve the user experience. If you notice many people asking similar questions, this is usually an indication that the documentation is weak on that point. Fix it. When the question is asked again, point to the improved documentation, and ask for feedback as to how it could be further improved.
+
+Take criticism of the documentation as a challenge, rather than a personal criticism.
+
+# Review PRs and tickets
+Dig into unmerged PRs and outstanding tickets. Figure out how to navigate the process to get an old ticket resolved, or a PR merged.
+
+This is an investment in the project in two ways.
+
+The obvious improvement is getting an issue fixed or a PR merges, thus enhancing the project. The less obvious way is that ancient tickets, and unmerged PRs, communicate that the project is not actively maintained, and that user issues are being ignored. This undermines project trust. Thus, addressing these things builds community trust, and increases your personal value to the project.
+
+# Be visible
+Participate meaningfully in discussions on the developer mailing list. Drive discussions through to action. Advocate for changes that help yourself and your employer, but also for those that improve the project as a whole.
+
+Get to know the important characters on the project, and what their priorities are. Help them achieve those priorities in whatever way you can. Figure out who you can best collaborate with to advance your own personal interests in the project, but also the overall health of the project and community.
+
+Start conversations around topics you’re passionate about, and volunteer to be the one to address them.
+
+Do not, however, just talk to be seen. Nobody is fooled by that.
+
```

Comment:
# Review/Improve Continuous Integration testing
Many projects use CI to continuously test proposed patches and PRs. Each platform needs to be individually set up for testing, and often the projects do not have enough people familiar with the specific environments that might benefit from testing. You can propose additional platforms to  test on